### DISCUSSION TITLE
Jump to message

### DISCUSSION THREAD

**Discussion by neil-lobo** [Chatterino/chatterino2#3161] [Ideas] [answered]
I've found myself on multiple occurrences clicking on a user and clicking on one of their messages to try jumping to it in chat (probably from discord muscle memory). I think it'd be a cool feature to add something like this :)

**Comment by xHeaveny** [comment]
i like that idear

**Comment by neil-lobo** [comment]
Just noticed #3054.

**Comment by Nerixyz** [comment, marked as answer]
Do you think #3953 solves your problem?
